src/compiler/scala/tools/nsc/transform/Erasure.scala

```diff
@@ -934,17 +950,24 @@ abstract class Erasure extends InfoTransform
      */
     private def bridgeDefs(owner: Symbol): (List[Tree], immutable.Set[Symbol]) = {
       assert(phase == currentRun.erasurePhase, phase)
-      new ComputeBridges(unit, owner) compute()
+      new GenerateBridges(unit, owner).generate(includeDeferred = false)
     }
 
-    def addBridges(stats: List[Tree], base: Symbol): List[Tree] =
+    def addBridgesToTemplate(stats: List[Tree], base: Symbol): List[Tree] =
       if (base.isTrait) stats
       else {
         val (bridges, toBeRemoved) = bridgeDefs(base)
         if (bridges.isEmpty) stats
         else (stats filterNot (stat => toBeRemoved contains stat.symbol)) ::: bridges
       }
 
+    def addBridgesToLambda(lambdaClass: Symbol): Unit = {
+      assert(phase == currentRun.erasurePhase, phase)
+      assert(lambdaClass.isClass, lambdaClass)
+      val eb = new EnterBridges(unit, lambdaClass)
+      eb.computeAndEnter(includeDeferred = true)
```

Comment:
Could you add a comment on the reason for `includeDeferred = true` part? It isn't clear to me.

Reply:
Ugh. I recall needing this, but I didn't comment it and now I can't remember. I'll poke at it until I find out why.

Reply:
Ah. I think it is that at one point I was cloning the SAM's `MethodSymbol` and not changing its flags, so it would show up deferred here. Good catch.